Library/Homebrew/utils.rb

```diff
@@ -454,6 +454,47 @@ def redirect_stdout(file)
     out.close
   end
 
+  # Ensure the given formula is installed
+  # This is useful for installing a utility formula (e.g. `shellcheck` for `brew style`)
+  def ensure_formula_installed!(formula_or_name, reason = "", latest: false, linked: false,
+                                output_to_stderr: false, quiet: false)
+    if output_to_stderr || quiet
+      file = if quiet
+        File::NULL
+      else
+        $stderr
+      end
+      # Call this method itself with redirected stdout
+      redirect_stdout(file) do
+        return ensure_formula_installed!(formula_or_name, reason, latest: latest, linked: linked)
+      end
+    end
+
+    require "formula"
+
+    formula = if formula_or_name.is_a?(Formula)
+      formula_or_name
+    else
+      Formula[formula_or_name]
+    end
+
+    reason = " #{reason}" if reason.present? # add a whitespace
```

Comment:
```suggestion
    reason = "for #{reason}" if reason.present? # add a whitespace
```

Reply:
I add `"for"` in the argument list where this method is called from. Not all `reason` are started with `"for"`.

Reply:
@XuehaiPan I think they could all be reworded to use `for`. Happy to help with suggestions there.